office-seat-lottery/src/components/form/form-radio.jsx

```
@@ -8,7 +8,7 @@ export function FormRadioGroup({ control, name, label, options }) {
       name={name}
       render={({
         field: { onChange, value },
-        fieldState: { error },
+        fieldState: {},
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)